### DISCUSSION TITLE
Wildfire resources

### DISCUSSION THREAD

**Discussion by peterkobylarz** [earthlab-education/Earth-Analytics-2023-01-Intro#78] [Resources]
Hi everyone,

Wildfire preparation, response, and resilience is one of my primary professional areas of focus. Wildfire has also been a near constant presence in my personal life over the last 6 years while living in the Bay Area and the Denver area. Here are a couple resources I use to monitor active fires, get notifications for areas I'm interested in, and download resources:

- Western Fire Chiefs Alliance map: https://wfca.com/fire-map
- Watch Duty: An awesome app that provides live info about fires and allows you to enable regional push notifications: https://www.watchduty.org/
- National Interagency Wildland Fire Open Data: this is a great resource for authoritative spatial data relating to wildfires in the US: https://data-nifc.opendata.arcgis.com/
- Fire Adapted Colorado: an awesome network of wildfire professionals that provides tons of resources, information and training: https://fireadaptedco.org/

Is anyone else involved in wildfire resiliency work? I'd love to meet and network with anyone else involved or interested in this area!

**Comment by edchanonline** [comment]
@peterkobylarz Love this - didn't know there are these resources. I subscribed to the CU alerts and [BOCO alerts](https://member.everbridge.net/453003085612231/login). Earth analytics side - I am still figuring out where I can contribute. In the Lafayette area and would love to connect!

**Comment by advythr** [comment]
@peterkobylarz Thanks for the resources! I do not have any professional experience in this area, but have been monitoring fires as similarly they have been a part of my personal life, growing up in the Bay Area and going to school in Southern California. I am interested in fire ecology and am hoping to study grassland fire as a PhD student in ecology at CU. Would be happy to learn more from you about this area!

**Comment by eculler** [comment]
Thanks, @peterkobylarz ! For anyone who is interested in wildfire, if you don't know this already we have a number of wildfire ecologists at Earthlab, led by our former director Dr. Jennifer Balch. I'd recommend keeping an eye on [our seminar](https://earthlab.colorado.edu/seminar) for related talks. We also have GRA or postdoc openings in the area pretty regularly. In particular, I'd recommend keeping an eye out for our summer graduate internships if you are admitted as a CU main campus graduate student.

**Reply by edchanonline** [reply]
This is great. Thank you!!
